src/devices/video/ppu2c0x_vt.h

```diff
@@ -0,0 +1,78 @@
+// license:BSD-3-Clause
+// copyright-holders:David Haywood
+/******************************************************************************
+
+    VT video emulation
+	
+	The VT video is based on the ppu2c0x but with enhanced capabilities such
+	as 16 colour sprites.
+
+******************************************************************************/
+
+#include "emu.h"
+#include "video/ppu2c0x.h"
+
+#define MCFG_PPU_VT03_ADD(_tag)   \
+	MCFG_PPU2C0X_ADD(_tag, PPU_VT03)
+
+#define MCFG_PPU_VT03_READ_BG_CB(_devcb) \
+	devcb = &ppu_vt03_device::set_read_bg_callback(*device, DEVCB_##_devcb);
+
+#define MCFG_PPU_VT03_READ_SP_CB(_devcb) \
+	devcb = &ppu_vt03_device::set_read_sp_callback(*device, DEVCB_##_devcb);
+
+class ppu_vt03_device : public ppu2c0x_device {
+public:
+	ppu_vt03_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock);
+
+	template<class _Object> static devcb_base &set_read_bg_callback(device_t &device, _Object object) { return downcast<ppu_vt03_device &>(device).m_read_bg.set_callback(object); }
```

Comment:
Please don't use names beginning with an underscore followed by a capital letter.  Also please make these follow the pattern of taking a universal reference and forwarding it, to support callback objects that are movable but not copyable, and callback objects that are more efficient to move (see any of the hundreds of device headers I updated).  Same applies to the ones added in the other header.